runtime/CSharp/Antlr4BuildTasks/Antlr4ClassGenerationTask.cs

```diff
@@ -207,8 +207,16 @@ private void ProcessBuildMessage(BuildMessage message)
             errorCode = Log.ExtractMessageCode(message.Message, out logMessage);
             if (string.IsNullOrEmpty(errorCode))
             {
-                errorCode = "AC1000";
-                logMessage = "Unknown build error: " + message.Message;
+                if (message.Message.StartsWith("Executing command:") && message.Severity == TraceLevel.Info)
```

Comment:
I suggest to use StringComparison.Ordinal for performance reasons. You don't really want to do locale-aware string comparsions in such code.